content/en/database_monitoring/connect_dbm_and_apm.md

```diff
@@ -194,7 +194,7 @@ In your Gemfile, install or update [dd-trace-rb][1] to version `1.8.0` or greate
 
 ```rb
 source 'https://rubygems.org'
-gem 'ddtrace', '>= 1.8.0'
+gem 'ddtrace', '>= 1.8.0' # Use 'datadog' if you're using v2.x
```

Comment:
```suggestion
gem 'datadog' # Use `'ddtrace', '>= 1.8.0'` if you're using v1.x
```

Reply:
Database monitoring support works correctly in 2.0, so that's recommend the clients use the latest version.